test/helpers/view_components/student/concepts_map_test.rb

```diff
@@ -0,0 +1,17 @@
+require_relative "../view_component_test_case"
+
+class Student::ConceptsMapTest < ViewComponentTestCase
+  test "component with empty concepts map rendered correctly" do
+    data = {
+      concepts: [],
+      layout: [],
+      connections: []
+    }
+    component = ViewComponents::Student::ConceptsMap.new(data).to_s
+
+    # TODO: Question: What is the second argument of this function supposed to be?
+    #   A hash representing the html structure?
+    assert_component_equal component,
+      { id: "concepts-map", props: { data: data } }
```

Comment:
Yeah - take a look in the `view_component_test_case.rb` file to see the definition.

We should probably actually just split this into three params now:
- component
- id
- props

I don't think the second needs to be a hash anymore